app/models/role_permission.rb

```diff
@@ -5,4 +5,15 @@ class RolePermission < ApplicationRecord
   belongs_to :role
 
   validates :value, presence: true
+
+  def self.room_limit_reached(user_id)
+    return render_error status: :not_found unless User.find(user_id)
```

Comment:
Only issue i find with this check per Amir's request, is that we are doing the User.find query twice now. Is this an issue? Is there a better way?

Reply:
Also can/will move this function to be part of PermissionChecker service

Reply:
I think you lost me on this,
I meant to maybe make a class method that reassembles to `MeetingOption.get_setting_value` that removes the need to write that query manually each time we fetch the value of a permission for a given role,